example/data-persistence-tf/variables.tf

```diff
@@ -62,6 +62,12 @@ variable "rds_user_password" {
   default = ""
 }
 
+variable "rds_connection_heartbeat" {
```

Comment:
I agree with the db connection object -- I was thinking this one would be separate from that, because this really is a DB diagnostic setting that's trying to get those timing log lines, and less about configuring the Knex connection itself. 

  If you (or others) feel strongly that the heartbeat boolean should be part of that object, that's a change I'm willing to and would rather make now than later.

Reply:
OK, let's call it YAGNI for now